### ISSUE TITLE
synfig project file broken after adding skeleton

### ISSUE THREAD

**Issue by Neimed** [synfig/synfig#1387]
On Synfig 1.3.1, same with 1.2.1, Ubuntu Studio 18.10

I can open a file after create skeleton, bones and children bones and link their to independents groups
this error message appears
```
loadcanvas.cpp:1653
/path/project.sif:<bone_valuenode>:65535: error: Element <bone_valuenode> unexpected
/path/project.sif:<bone_valuenode>:65535: error: bad data in <bone_valuenode>
loadcanvas.cpp:2624 parse_value_node done bad data
/path/project.sif:<bone_valuenode>:65535: error: Parsing of "bone" failed
/path/project.sif:<bone_link>:65535: error: <bone_link> is missing link 0 (bone)
...
```
(message translate from the french)
and this for every bunch of bones

have a nice day

**Comment by rodolforg** [comment]
@Neimed Could you post your file?
Or describe how did you create your skeleton and its bones?

**Comment by Neimed** [comment]
I finally found the solution. 
Here is the manipulation: in a project A I created a layer linked to a skeleton. Then I copied it to a project B then I deleted the skeleton from the layer in project B. The links to the frame stay and therefore is link to non-existent bones.

So I opened the corrupted sif file with a text editor and I looked for all the "bone_link" which had a nonexistent reference in this project, I replaced the bone_link paragraph by a composite paragraph for example (i used an already existing composite paragraph for syntax 😅) and voilà ! 
Thanks for all
